src/applications/disability-calculator/utils/helpers.js

```diff
@@ -1,6 +1,6 @@
 // helper function to round to nearest 10
 function roundRating(num) {
-  const ratingWithDecimal = Number.parseFloat(num).toFixed(0.1);
+  const ratingWithDecimal = parseFloat(num).toFixed(0.1);
```

Comment:
Untested in IE, but I suspect this will fix this IE issue, https://github.com/department-of-veterans-affairs/va.gov-team/issues/428.